distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -94,6 +94,10 @@ type Point_Data
             candidates.find if_missing=candidates.first is_good
         Point_Data.Row_Number ->
             Point_Data.iota table.row_count
+        Point_Data.Color ->
+            colour_col = table.lookup_ignore_case "Colour"
+            is_good c = (self.is_recognized c).not
+            if is_good colour_col then colour_col else Error.throw Nothing
```

Comment:
Looks much better.

Btw. not directly related to this change but I see that in this method, if the column is not found, we sometimes `throw Nothing` and sometimes throw `No_Fallback_Column`. Signature suggests `No_Fallback_Column`, so if possible would be good to get this consistent (although tbh it's not that important as long as it doesn't break anything since it's just internal code - still would be good to make it clearer if possible).